### PR TITLE
OSDOCS-2691 Release Notes for 4.10

### PR DESCRIPTION
**Pull request by sagidlow**
Release notes for the new Getting Started section that is documented per JIRA: https://issues.redhat.com/browse/OSDOCS-2691. 
Once related PR(https://github.com/openshift/openshift-docs/pull/39254) is merged, we'll add the link to the Getting Started and merge the PR. 

SME: @sjstout needs to sign off before merging. 
No QE is needed. 

Preview: [Getting Started with OpenShift Container Platform](https://deploy-preview-41964--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-documentation).

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -42,6 +42,25 @@ The scope of support for layered and dependent components of {product-title} cha
...
 
+[id="ocp-4-10-documentation"]
+=== Documentation
+
+[id="ocp-4-10-getting-started"]
+==== Getting started with {product-title}
+
+{product-title} 4.10 now includes a getting started guide. Getting Started with {product-title} defines basic terminology and provides role-based next steps for developers and administrators.
+
+The tutorials walks new users through the web console and the OpenShift CLI (`oc`) interfaces. New users accomplish the following tasks through the Getting Started.
```
Comment (sjstout) [inline release_notes/ocp-4-10-release-notes.adoc:53]: typo: should be "The tutorials walk" without the s in walks

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -42,6 +42,25 @@ The scope of support for layered and dependent components of {product-title} cha
...
+* Deploy a container image
+* Add a Python application
+* Scale applications
+* Create routes and grant view permissions
+* Connect to a database, load, and view your application.
+
+// For more information, see Getting Started with {product-title}
+
 [id="ocp-4-10-rhcos"]
 === {op-system-first}
```
Comment (sjstout) [inline release_notes/ocp-4-10-release-notes.adoc:66]: Other than the 1 minor typo, LGTM! Thanks for adding this!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -42,6 +42,25 @@ The scope of support for layered and dependent components of {product-title} cha
...
 
+[id="ocp-4-10-documentation"]
+=== Documentation
+
+[id="ocp-4-10-getting-started"]
+==== Getting started with {product-title}
+
+{product-title} 4.10 now includes a getting started guide. Getting Started with {product-title} defines basic terminology and provides role-based next steps for developers and administrators.
+
+The tutorials walk new users through the web console and the OpenShift CLI (`oc`) interfaces. New users accomplish the following tasks through the Getting Started.
```
Comment (mburke5678) [inline release_notes/ocp-4-10-release-notes.adoc:53]: s/users/users can ??

Do you need a colon before the list?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 4]
```diff
@@ -42,6 +42,25 @@ The scope of support for layered and dependent components of {product-title} cha
...
+
+* How to create a project
```
Comment (mburke5678) [inline release_notes/ocp-4-10-release-notes.adoc:55]: Should this be  _Create a project_? I reckon you are trying to mirror the headings in the Getting Started, but it seems odd to have a _How to_ in the list.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 5]
```diff
@@ -42,6 +42,25 @@ The scope of support for layered and dependent components of {product-title} cha
...
+* Deploy a container image
+* Add a Python application
+* Scale applications
+* Create routes and grant view permissions
+* Connect to a database, load, and view your application.
```
Comment (mburke5678) [inline release_notes/ocp-4-10-release-notes.adoc:60]: `Connect to a database, load, and view your application.`
This seems wrong. This means  Connect to a database, connect to a load, and connect to a view your application. Not sure what the fix is. 
`Connect to a database to load and view your application.`
`Connect to a database, then load and view your application.`

Comment (sagidlow) [reply release_notes/ocp-4-10-release-notes.adoc:60]: @mburke5678 My apologies - in a rush, I left out the word "load data". Does this sound better?

Connect to a database, load your data, and view your application.

**Comment by mburke5678** [comment]
@sagidlow  A couple of nits. Otherwise LGTM.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 6]
```diff
@@ -42,6 +42,27 @@ The scope of support for layered and dependent components of {product-title} cha
...
+==== Getting started with {product-title}
+
+{product-title} 4.10 now includes a getting started guide. Getting Started with {product-title} defines basic terminology and provides role-based next steps for developers and administrators.
+
+The tutorials walk new users through the web console and the OpenShift CLI (`oc`) interfaces. New users can accomplish the following tasks through the Getting Started:
+
+* Create a project
+* Grant view permissions
+* Deploy a container image from Quay
+* Examine and scale application
```
Comment (ctauchen) [inline release_notes/ocp-4-10-release-notes.adoc:58]:
```suggestion
* Examine and scale an application
```
Just to keep in parallel with other list items

**Review by ctauchen — COMMENTED** [review]
One small thing. LGTM!

**Pull request merged**